shared/notifications/index.ts

```diff
@@ -5,10 +5,6 @@
  * into numbers in only one place. This should make merge conflicts easier.
  */
 export const NOTIFICATION_DROP_LEDGER_FIREFOX = 25;
-export const NOTIFICATION_U2F_LEDGER_LIVE = 28;
-export const NOTIFICATION_BLOCKAID_DEFAULT = 29;
```

Comment:
Should all instances of this const be removed from the cod base? It's still present in tests

![Screenshot 2024-06-11 at 8 39 05 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/345982d3-0bf7-434a-aab4-e9ae8c140b14)

Reply:
nice find. These should be fixed on the develop branch